src/Microsoft.AspNet.JsonPatch/Helpers/GetValueResult.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.JsonPatch.Helpers
+{
+    /// <summary>
+    /// Return value for the helper method used by Copy/Move.  Needed to ensure we can make a different
+    /// decision in the calling method when the value is null because it cannot be fetched (HasError = true) 
+    /// versus when it actually is null (much like why RemovedPropertyTypeResult is used for returning 
+    /// type in the Remove operation).
+    /// </summary>
+    internal class GetValueResult
```

Comment:
We don't use internal unless there's a very good reason. This is already in an out-of-the-way namespace, just make it public.